src/api/tests/migrations/test_fresh_mysql_upgrade.py

```
@@ -0,0 +1,172 @@
+
+
+def test_fresh_mysql_upgrade_reaches_head():
+    base_uri = _get_base_mysql_uri()
+    temp_uri, database_name = _create_temp_database(base_uri)
+    expected_head = _get_expected_head()
+
+    env = os.environ.copy()
+    env["SQLALCHEMY_DATABASE_URI"] = temp_uri
+
+    try:
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)